expenses-manager-backend/src/main/java/com/endava/expensesmanager/controller/UsersController.java

```diff
@@ -0,0 +1,49 @@
+package com.endava.expensesmanager.controller;
+
+import com.endava.expensesmanager.entity.Users;
+import com.endava.expensesmanager.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/users")
+public class UsersController {
+
+    @Autowired
+    private UsersService usersService;
```

Comment:
aici ai facut Field Injection, iti recomand sa folosesti Constructor Injection. Aici poti sa gasesti mai multe detalii despre tipurile de injectare si care sunt avantajele/dezavantajele  https://medium.com/@anil.java.story/why-spring-constructor-injection-is-the-recommended-approach-75edca1f9b36